keda/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Event-based autoscaler for workloads on Kubernetes
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.4.0
+version: 2.5.0
```

Comment:
Can you revert the new version please?

Reply:
Ah damn!

Reply:
It's best to just revert this change along with the new version & feed update.

We typically do this separately as part of a release (which is coming soon)